models/action_recognition/model_templates/gesture-recognition/s3d-rgb-mobilenet-v3-stream-msasl/template.yaml

```diff
@@ -22,6 +22,8 @@ dependencies:
   destination: packages/ote
 - source: ../../requirements.txt
   destination: requirements.txt
+- source: ../tools
```

Comment:
Still, why did you decide to make a part of instantiated template?

Reply:
To shortcut the way to data tools for the user.